tools/esdlcmd/esdl-publish.cpp

```diff
@@ -1132,13 +1134,14 @@ class EsdlBindLogTransformCmd : public EsdlBindServiceCmd
     void usage()
     {
         printf( "\nUsage:\n\n"
-                "esdl bind-log-transform <TargetESDLBindingID> <TargetLogTransformName> --config <file|xml> [command options]\n\n"
+                "esdl bind-log-transform <TargetESDLBindingID> [TargetLogTransformName] --config <file|xml> [command options]\n\n"
                 "   TargetESDLBindingID                              The id of the target ESDL binding (must exist in dali)\n"
-                "   TargetLogTransformName                           The name of the target LogTransform (must exist in the service ESDL definition)\n"
+                "   TargetLogTransformName                           The name of the target LogTransform (required when config is xsl transform)\n"
                 "   --config <file|xml>                              Configuration XML for all LogTransforms associated with the target Service\n"
 
                 "\nOptions (use option flag followed by appropriate value):\n"
-                "   --overwrite                                      Overwrite binding if it already exists\n");
+                "   --overwrite                                      Overwrite binding if it already exists\n"
```

Comment:
Probably should have seen this before, but better late than never... The binding is assumed to exist, so --overwrite applies to the transform(s) in question.

Reply:
Will fix